packages/apps-config/src/settings/endpoints.ts

```diff
@@ -124,8 +124,8 @@ function createTest (t: TFunction): LinkOption[] {
     },
     {
       info: 'edgeware',
-      text: t<string>('rpc.berlin', 'Berlin (Edgeware Testnet, hosted by Commonwealth Labs)', { ns: 'apps-config' }),
-      value: 'wss://berlin1.edgewa.re'
+      text: t<string>('rpc.beresheet', 'Beresheet (Edgeware Testnet, hosted by Commonwealth Labs)', { ns: 'apps-config' }),
+      value: 'ws://beresheet1.edgewa.re:9944'
```

Comment:
This needs to b `wss` - the issue is that https cannot connect to non-secure ws.